mlir/tools/mlir-tblgen/AttrOrTypeDefGen.cpp

```diff
@@ -30,7 +30,7 @@ using namespace mlir::tblgen;
 /// Find all the AttrOrTypeDef for the specified dialect. If no dialect
 /// specified and can only find one dialect's defs, use that.
 static void collectAllDefs(StringRef selectedDialect,
-                           std::vector<llvm::Record *> records,
+                           const std::vector<const llvm::Record *> &records,
```

Comment:
Can you just switch this to ArrayRef now?

Reply:
Done.